lib/Conversion/LlvmToNeura/LlvmToNeuraPass.cpp

```diff
@@ -134,6 +134,42 @@ struct LlvmSRemToNeuraRem : public OpRewritePattern<LLVM::SRemOp> {
   }
 };
 
+struct LlvmMaxNumToNeuraFMax : public OpRewritePattern<LLVM::MaxNumOp> {
+  using OpRewritePattern::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(LLVM::MaxNumOp op,
```

Comment:
LLVM::MaxNumOp is different LLVM::MaximumOp, can we add attribute to neura.max indicating which one is lowered from? You can also add test for `neura.max()<maxnum>` and `neura.max()<maximum>` when `NAH` is serving as operand.